### DISCUSSION TITLE
Database for multiple apps

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7380] [General]
I want to use same database for multiple apps, how can I achieve this

**Comment by eyduh** [comment]
Hi,

With external databases (Postgres/MySQL/etc) you jsut need to connect the apps and fetch the tables.

To use the same budibase db I'm not sure how to achieve this other than some hacky automation to keep different tables in sync.

**Reply by melohagan** [reply]
It's worth noting as well that you can connect to an external Couch DB instance (which is the same type of database used by the Budibase DB):

![Screenshot 2022-08-22 at 13 45 37](https://user-images.githubusercontent.com/101575380/185924282-4fadbbb9-19f7-465e-9aa6-895c5b573d46.png)

**Reply by sergioponguta** [reply]
> It's worth noting as well that you can connect to an external Couch DB instance (which is the same type of database used by the Budibase DB):
> 
> ![Screenshot 2022-08-22 at 13 45 37](https://user-images.githubusercontent.com/101575380/185924282-4fadbbb9-19f7-465e-9aa6-895c5b573d46.png)

Doing so you are not going to have the advantages of Budibase DB right?

Do you have any idea in how can I share the Budibase DB between two apps? Or any workaround? It's the only thing i'm missing now. 😢

**Comment by JCV67** [comment]
Would be very helpful to have some kind of a supraset of tables that would be always available to all apps. Tables like employees, cars, etc. that a are needed in different apps, just like users table.
